src/sentry/api/event_search.py

```diff
@@ -1417,7 +1417,7 @@ def get_json_meta_type(field_alias, snuba_type, function=None):
     return snuba_json
 
 
-FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>[^\)]*)\)$")
+FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>[^\)]*)\)( as (?P<alias>\w+))?$")
```

Comment:
Should we allow `AS`. Also right now `as as` is valid as is `as 123`, should they be?

Reply:
:+1: to allowing `AS`, 🤔 tbh I don't see a problem with `f() as as`, is there a reason we'd want to avoid that?

hmm but looks like snuba is turning `count() as 123` to `count() as _snuba_123` but even then I'd prefer to avoid digit only aliases.